tests/scripts/travis_scripts.sh

```diff
@@ -1,15 +1,40 @@
 #!/bin/bash
 
 # Common checks to get run during the 'script' section in Travis.
+OUTPUT=0
+
+# Make OUTPUT equal return code if return code is not 0
+function checkReturn {
+  if [ $1 -ne 0 ]; then
+    OUTPUT=$1
+  fi
+}  
 
 # Lint
 find $TRAVIS_BUILD_DIR -type f \( -name '*.php' -o -name '*.inc' -o -name '*.module' -o -name '*.install' -o -name '*.test' \) -exec php -l {} \;
```

Comment:
Since now I'm thinking if each of these actually test anything, this line doesn't,`find` here won't ever return anything but 0 even if a lint fails. I know it was preexisting, but might be good to either fix or remove. 
https://apple.stackexchange.com/questions/49042/how-do-i-make-find-fail-if-exec-fails